backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraZonedDateTimeModule.java

```diff
@@ -22,41 +22,25 @@
 import static com.datastax.driver.core.DataType.text;
 import static com.datastax.driver.core.DataType.timestamp;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.apache.james.backends.cassandra.components.CassandraModule;
-import org.apache.james.backends.cassandra.components.CassandraTable;
-import org.apache.james.backends.cassandra.components.CassandraType;
 
 import com.datastax.driver.core.schemabuilder.SchemaBuilder;
 
-public class CassandraZonedDateTimeModule implements CassandraModule {
+public class CassandraZonedDateTimeModule extends CassandraModuleComposite {
```

Comment:
when you think about it, this inheritance is a bit weird : "my module is a composite".
It's not.
When the only useful line is calling a constructor, then you don't need the class at all.

Reply:
If you wish so.

Reply:
I agree with @mbaechler. extending `CassandraModuleComposite` cause a confusing to me, its implementation doesn't composite modules but just one module

Reply:
Sure me also I agree.

That gonna make a wonderful 1.000 + LOC diff as a fixup and will take me a few hours to come up with...

Reply:
Also before polishing things, and investing a huge time refactoring, I like to gain some consensus before, in order to not waist my time.

Reply:
> That gonna make a wonderful 1.000 + LOC diff as a fixup and will take me a few hours to come up with...

Can we stop with this kind of answer ?